models/multiphase/d3q27_pf_velocity/Boundary.c.Rt

```diff
@@ -70,42 +70,127 @@ CudaDeviceFunction void <?R C(my_velocity_boundaries[ii])?>(){
 		C(h[sel], heq[sel] + heq[bounce][sel] - h[bounce][sel] - 0.5 * Unknowns * (exM))
 	?>
 }
+<?R
+}
+?>
 
-CudaDeviceFunction void <?R C(my_pressure_boundaries[ii])?>(){
+<?R
+for (ii in 1:cube_faces){
+for (j in 1:length(pressure_boundary_types)) {  ?>
+CudaDeviceFunction void <?%s paste0(C(my_pressure_boundaries[ii]), pressure_boundary_types[j]) ?>(){
     real_t d = getRho();
```

Comment:
@shkodm should d be based off a prescribed phase on the boundary rather than from getRho - which I believe calls from PhaseF field

Reply:
yea, it is then set properly in the if condition below

Reply:
I did see this, but figured it was maybe not best to define it with a value knowing it will be overwritten. I was also curious how this worked with the open condition - but perhaps I need to just look at the code a bit deeper. I will try to do this before tomorrow when we meet but have a bit of a marking pile with end of semester.